backend/util/environment.go

```diff
@@ -41,6 +44,6 @@ func IsBackendInDocker() bool {
 	return InDockerGo == "true"
 }
 
-func IsOnRender() bool {
-	return DopplerConfig == "prod_aws_render"
+func IsProduction() bool {
+	return strings.HasPrefix(DopplerConfig, "prod")
```

Comment:
So not sure if this is relevant, but I believe some of the errors were linked to doppler configs that used docker, so would these errors still be in our project? Should they be?
<img width="588" alt="Screenshot 2023-08-28 at 11 29 43 AM" src="https://github.com/highlight/highlight/assets/17744174/9a0727bc-ffaa-4449-853b-c80d29cb6523">

Reply:
the `IsProduction` check is just to make sure we apply this filtering logic only for our production deploy. The reason for that is simply that only our production cloud deploy is affected by this problem (where other external errors are sent to us), whereas other self-hosted stacks don't need to drop any errors from project 1.